### PR TITLE
Added the new switch parameter IdentifyTopFatFlows to Firewall Command

### PR DESCRIPTION
**Pull request by Gizachew-Eshetie**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from the scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** change version of module in pull request

### CODE REVIEW

File: src/Network/Network/help/Set-AzFirewall.md [hunk 1]
```diff
@@ -165,11 +167,22 @@ $Hub = Get-AzVirtualHub -ResourceGroupName "testRG" -Name "westushub"
...
 Set-AzFirewall must be called.
 
+### 13:	Identify Top Fat Flows on Azure Firewall
+```powershell
+$azFw = Get-AzFirewall -Name "AzureFirewall" -ResourceGroupName "rg"
+$azFw.IdentifyTopFatFlows = $true
+
+$azFw | Set-AzFirewall
+```
+
```
Comment (BethanyZhou) [inline src/Network/Network/help/Set-AzFirewall.md:183]: Could you provide sample output with following format
\```output
{output}
\```
If this example has no output, please fill it with `None`.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/Network/Network/AzureFirewall/NewAzureFirewallCommand.cs [hunk 2]
```diff
@@ -232,6 +232,12 @@ public class NewAzureFirewallCommand : AzureFirewallBaseCmdlet
         )]
         public SwitchParameter AllowActiveFTP { get; set; }
 
+        [Parameter(
+           Mandatory = false,
+           HelpMessage = "Identify Top Fat Flows. By default it is false."
+       )]
+        public SwitchParameter IdentifyTopFatFlows { get; set; }
```
[the remark this answers is not in the document]
Comment (Gizachew-Eshetie) [reply src/Network/Network/AzureFirewall/NewAzureFirewallCommand.cs]: @BethanyZhou made changes and pushed. Please let me know if you have any more comments.

File: src/Network/Network/help/Set-AzFirewall.md (hunk 1 above)
Comment (Gizachew-Eshetie) [reply src/Network/Network/help/Set-AzFirewall.md:183]: @BethanyZhou Added the output and pushed.

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Pull request merged**